### PR TITLE
fix: browser compatibility issues

### PR DESCRIPTION
**Pull request by iMyth**

### CODE REVIEW

**Comment by iMyth** [comment]
#318

File: src/component.ts [hunk 1]
```diff
@@ -109,6 +109,13 @@ const reservedPropertyNames = [
   'filter'
 ]
 
+const shouldIgnore = {
+  'prototype': true,
+  'arguments': true,
+  'callee': true,
+  'caller': true
```
Comment (ktsn) [inline src/component.ts]: Can we remove the quotes?

File: src/component.ts [hunk 2]
```diff
@@ -117,7 +124,7 @@ function forwardStaticMembers (
   // We have to use getOwnPropertyNames since Babel registers methods as non-enumerable
   Object.getOwnPropertyNames(Original).forEach(key => {
     // `prototype` should not be overwritten
```
Comment (ktsn) [inline src/component.ts]: Let's update this comment 🙂

File: src/component.ts (hunk 1 above)
Comment (iMyth) [reply src/component.ts]: Sure, quotes are removed, thank you for your kind advice!

File: src/component.ts (hunk 2 above)
Comment (iMyth) [reply src/component.ts]: Done, thank you 🙂

**Pull request merged**